source/_integrations/kaleidescape.markdown

```diff
@@ -35,6 +37,33 @@ This integration is intended for the automation of Kaleidescape players with a m
 
 The Kaleidescape media player platform will create a Media Player entity for the device. This entity will display the currently playing media and playback controls.
 
+## Remote
+
+The Kaleidescape remote platform will create a Remote entity for the device. This entity allows you to send the following commands via the `remote.send_command` service.
```

Comment:
Let's link the `remote.send_command` service to the appropriate documentation,

Reply:
Fixed in https://github.com/home-assistant/home-assistant.io/pull/22006.